pypeit/spectrographs/vlt_xshooter.py

```diff
@@ -444,6 +416,8 @@ def check_frame_type(self, ftype, fitstbl, exprng=None):
                                 | (fitstbl['target'] == 'LAMP,FLAT'))
                                & good_dark_seq)
         
+        # TODO: Figure out how to identify lampoffflats
```

Comment:
You could follow what @debora-pe did for MOSFIRE.

Reply:
I had forgotten that, actually, the lampoffflats were already being identified, but mistyped as dark frames (see the code block immediately above). So it was even easier than I was expecting!